System/Console/ANSI/Codes.hs

```diff
@@ -60,13 +60,18 @@ module System.Console.ANSI.Codes
   , setTitleCode
 
     -- * Utilities
-  , colorToCode, csi, sgrToCode
+  , colorToCode
```

Comment:
This may be an @feuerbach level decision: I am also wondering (see below) if this module should still export `colorToCode` if module `System.Console.ANSI.Color` is to export that function and be exposed. That would be a breaking change from the past.

Reply:
@mpilgrem @feuerbach, I'll leave it as-is for now, since it's a breaking change. I'll just add a comment saying that in the future major release, this should be removed.